scripts/ci/make/util

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env bash
+
+#########################################################
+###   COMMON UTILITY FUNCTIONS - ALPHABETICAL ORDER   ###
+#########################################################
+
+cleanup() {
+    rm -rf "${VENV}/tmp"
+}
+
+clean_container() {
+    print_colorized ERROR "deleting ${TEST_IMAGE_NAME}"
+    sleep 3
+    docker rmi "${TEST_IMAGE_NAME}"
+}
+
+clean_python_env() {
+    print_colorized ERROR "deleting virtualenv & build data"
+    sleep 3
+    rm -rf "${VENV}" \
+        *.egg-info \
+        dist \
+        build
+    find . -name '*.pyc' -exec rm -f {} \;
+}
+
+clean_tests() {
+    print_colorized ERROR "deleting testing temp data"
+    sleep 3
+    rm -rf .tox
+    rm -f tox.log
+    rm -rf .pytest_cache
+}
+
+docker_compose_down() {
+    docker-compose -f scripts/ci/docker-compose-ci.yaml down
+}
+
+docker_compose_up() {
+    docker-compose -f scripts/ci/docker-compose-ci.yaml up -d
+}
+
+install() {
+    print_colorized WARN "installing ${GIT_REPO} to ${VENV}"
+    pip install .
```

Comment:
why is this using `pip install` ? I would prefer to use the more standard way with `python setup.py install`. This function is also saying it is going to install in a `${VENV}` but that is only true if `pip` is coming from the virtualenv, it isn't checking whether that is true or not